react.js

```diff
@@ -13,11 +23,19 @@ stuffArray.map(function(el, i){
 
 // your for loop here, you can use the same stuffArray array
 
+for(let i=0;i<stuffArray.length;i++){
+  console.log(stuffArray[i] + " is at index: "+ [i])
+}
 
-// 3. Basic sorting: Find the cheapest price -- you are given a list of prices and need to find the signle lowest price from the array. Write a function that takes in an array of numbers, and returns the lowest price
-
-var steal = function(){
-
-  // your logic here
-  
```

Comment:
This logic works for the array you created, but setting 'lowest' dynamically would give you more flexibility. Maybe starting with array[0]?